networkx/utils/backends.py

```diff
@@ -960,19 +959,33 @@ def _make_doc(self):
                 continue
 
             func_info = info["functions"][self.name]
-            if "extra_docstring" in func_info:
+
+            # Renaming extra_docstring to backend_func_docs
+            if ("extra_docstring" in func_info and func_info["extra_docstring"]) or (
+                "backend_func_docs" in func_info and func_info["backend_func_docs"]
+            ):
+                key = (
+                    "extra_docstring"
+                    if func_info.get("extra_docstring")
+                    else "backend_func_docs"
+                )
                 lines.extend(
-                    f"  {line}" if line else line
-                    for line in func_info["extra_docstring"].split("\n")
+                    f"  {line}" if line else line for line in func_info[key].split("\n")
```

Comment:
And also:
```suggestion
            if func_docs := (
                func_info.get("backend_func_docs") or func_info.get("extra_docstring")
            ):
                lines.extend(
                    f"  {line}" if line else line for line in func_docs.split("\n")
```
(feel free to play around to find syntax that you like)

Reply:
sry I wasn't able to do "commit suggestion" after i pushed some commits.